packages/dds/tree/src/api.ts

```diff
@@ -0,0 +1,20 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import type { TreeView, TreeViewAlpha, ImplicitFieldSchema } from "./simple-tree/index.js";
+
```

Comment:
Nit: We should leave a module-level comment here explaining the pattern we're going with here. Can probably extract the private remarks you have below into that comment.